app/views/shared/_worker_complete.html.erb

```diff
@@ -4,14 +4,10 @@
 <% continue_text = locals[:continue_text] || nil %>
 
 <%= page_template page_title: heading do %>
-  <p>
-    <%= intro_text %>
-  </p>
-  <p>
-    <%= balance_text %>
-  </p>
-  <div class="govuk-!-padding-top-3">
-    <%= next_action_link(continue_text:) %>
-  </div>
+  <p class="govuk-body"><%= intro_text %></p>
+  <p class="govuk-body"><%= balance_text %></p>
+
+  <%= govuk_button_link_to(continue_text, forward_path) %>
```

Comment:
does this need the `govuk-!-padding-top-3` still? as did this one `app/views/citizens/accounts/index.html.erb`

Reply:
@jsugarman I removed it intentionally because it was just a standard button below a standard paragraph so I thought it should have the default padding. Happy to put it back in if you think I should though